SPRCLI/Sources/SPR/SPRPublisher.swift

```diff
@@ -5,17 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import Foundation
-import ClientRuntime
-
```

Comment:
In the type below, the configuration code is removed.  This type now just publishes to SPR after the CLI commands have resolved all the parameters.